itest/performance/mint_batch_test.go

```diff
@@ -0,0 +1,193 @@
+package perf_test
+
+import (
+	"context"
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	_ "embed"
+
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/lightninglabs/taproot-assets/fn"
+	"github.com/lightninglabs/taproot-assets/itest"
+	"github.com/lightninglabs/taproot-assets/taprpc"
+	"github.com/lightninglabs/taproot-assets/taprpc/mintrpc"
+	unirpc "github.com/lightninglabs/taproot-assets/taprpc/universerpc"
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed testdata/8k-metadata.hex
+var imageMetadataHex []byte
+
+// execMintBatchStressTest checks that we are able to mint a batch of assets
+// and that other memebers in the federation see the universe updated
+// accordingly.
+func execMintBatchStressTest(t *testing.T, ctx context.Context, cfg *Config) {
+	// Create tapd clients.
+	alice, aliceCleanUp := getTapClient(t, ctx, cfg.Alice.Tapd)
+	defer aliceCleanUp()
+
+	bob, bobCleanUp := getTapClient(t, ctx, cfg.Bob.Tapd)
+	defer bobCleanUp()
+
+	// Create bitcoin client.
+	bitcoinClient := getBitcoinConn(t, cfg.Bitcoin)
+
+	imageMetadataBytes, err := hex.DecodeString(
+		strings.Trim(string(imageMetadataHex), "\n"),
+	)
+	require.NoError(t, err)
+
+	aliceHost := fmt.Sprintf("%s:%d", cfg.Alice.Tapd.Host,
+		cfg.Alice.Tapd.Port)
+
+	timeout := time.Duration(cfg.TestTimeout) * time.Minute
+	mintBatchStressTest(
+		t, ctx, bitcoinClient, alice, bob, aliceHost, cfg.BatchSize,
+		imageMetadataBytes, timeout,
+	)
+}
+
+func mintBatchStressTest(t *testing.T, ctx context.Context,
```

Comment:
Looks like this mostly copies the existing tests vs being able to re-use the same test scenario func?